### PR TITLE
Make 30702 message more clear.

### PR DESCRIPTION
**Pull request by antonsoroko**
## Description

for https://github.com/elgatito/plugin.video.elementum/issues/952

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Translation

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by elgatito** [comment]
@antonsoroko Error message is not mentioning login, because errors about limits are returned for both logged and anonymous users (with different descriptions and mentioned limits).

**Comment by antonsoroko** [comment]
@elgatito removed part about login. if you like this new message - then please merge, otherwise we can leave old message.

File: resources/language/Russian/strings.po [hunk 1]
```diff
@@ -2239,5 +2239,5 @@ msgid "English | en"
 msgstr "Английский | en"
 
 msgctxt "#30702"
-msgid "Exceeded daily allowed subtitles downloads"
-msgstr "Достигнуто максимальное дневное количество скачиваний субтитров"
+msgid "Reached daily limit for subtitles downloads"
+msgstr "Достигнут дневной лимит на загрузку субтитров без авторизации"
```
Comment (elgatito) [inline resources/language/Russian/strings.po:2243]: Leftovers

File: resources/language/messages.pot [hunk 2]
```diff
@@ -2250,5 +2250,5 @@ msgid "English | en"
 msgstr ""
 
 msgctxt "#30702"
-msgid "Exceeded daily allowed subtitles downloads"
+msgid "Reached daily limit for subtitles downloads without login"
```
Comment (elgatito) [inline resources/language/messages.pot:2253]: Leftovers

File: resources/language/Russian/strings.po (hunk 1 above)
Comment (antonsoroko) [reply resources/language/Russian/strings.po]: fixed

File: resources/language/messages.pot (hunk 2 above)
Comment (antonsoroko) [reply resources/language/messages.pot]: fixed.
also fixed lefover in 30703.
thanks.

**Pull request merged**